ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -290,6 +290,8 @@ TFLiteInterpreter::loadModel (bool use_nnapi)
   }
 
   interpreter->UseNNAPI (use_nnapi);
+  if (num_threads > 0)
```

Comment:
Is there no max limit?

Reply:
It seems no limit in tf-lite function. I added code to check hardware_concurrency() before setting the num_threads.